DopplerFiles/public-keys-dev/dev.xml

```diff
@@ -0,0 +1 @@
+<RSAKeyValue><Modulus>zUoxxxVc9cf3V05eTmT8GeKIojLl6ohzu2TDH5s4bQrJ7n5CIlUesnXuLmy5s/DnTLrD2WjK5rgjHE+oNT1bUbYCxRVaHsoC9THMjbWwEGOkurNcZ691RyymQmlW8485EbxWwJUUOuksJtqv3j5IqMWzLBaymACQ52zhJ2aI2dBZclAfTb+YpypDTwotYdLTw/YCCTRbuMH6ePabLroAqapVVdKmqZfIIJ0xOR813IQNKr2tXn938j1kB4ayDbFBCNEGqUb0gjOaplMvxJPqutesfMzxllepLV8pZvrEwg9Kj4FqpICfQgydZzoXfHfQa3I2mHGNByraom26xSL8TQ==</Modulus><Exponent>AQAB</Exponent></RSAKeyValue>
```

Comment:
What do you think about adding also the developers private key and the readme that exists in other projects?

Or at least a readme in this project with a reference to them.

I think that it will help new developers to understand the context.